code/game/mecha/mecha_wreckage.dm

```diff
@@ -32,6 +32,9 @@
 	if(AI)
 		user << "<span class='notice'>The AI recovery beacon is active.</span>"
 
+/obj/structure/mecha_wreckage/examine_status() // it's wreckage what more do you need to know
+	..()

```

Comment:
Errr I think you meant to just set this as the wreckage's description without the health stuff.